### PR TITLE
Added 2 Libraries from Yalantis

### PR DESCRIPTION
**Pull request by axelrindle**
Added [Context-Menu.Android](https://github.com/Yalantis/Context-Menu.Android) and [Side-Menu.Android](https://github.com/Yalantis/Side-Menu.Android) from [Yalantis](https://github.com/Yalantis/).

### CODE REVIEW

File: library/src/main/res/values/library_context_menu.android_strings.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<resources>
+	<string name="define_int_context_menu.android"></string>
```
Comment (mikepenz) [inline library/src/main/res/values/library_context_menu.android_strings.xml]: I don't think using a "." is a good idea. I haven't tried it to be honest. But til now there were non mit "."'s  :D

File: library/src/main/res/values/library_context_menu.android_strings.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+	<!-- Library section -->
+	<string name="library_context_menu.android_libraryName">Context-Menu.Android</string>
+	<string name="library_context_menu.android_libraryDescription">You can easily add awesome animated context menu to your app.</string>
+	<string name="library_context_menu.android_libraryWebsite">https://github.com/Yalantis/Context-Menu.Android</string>
+	<string name="library_context_menu.android_libraryVersion">1.0.7</string>
+	<!-- OpenSource section -->
+	<string name="library_context_menu.android_isOpenSource">true</string>
+	<string name="library_context_menu.android_repositoryLink">https://github.com/Yalantis/Context-Menu.Android</string>
+	<!-- ClassPath for autoDetect section -->
+	<string name="library_context_menu.android_classPath">com.yalantis.contextmenu.lib</string>
```
Comment (mikepenz) [inline library/src/main/res/values/library_context_menu.android_strings.xml]: please change the classpath to reference a specific class file, as it's not possible in java to check if a package exists. Just use one of the main classes. should be fine.(the same please also for the second lib)

**Comment by mikepenz** [comment]
@axelrindle have you tried the id with the "." (dots) in the name? I haven't yet had such a library and I am not sure if it will work

**Comment by axelrindle** [comment]
@mikepenz I changed the "." to a "_".

**Comment by mikepenz** [comment]
thanks.

**Comment by mikepenz** [comment]
but you didn't push :D seems to not yet be up

**Comment by axelrindle** [comment]
@mikepenz Actually I did.

**Comment by mikepenz** [comment]
I can't see the change in the diff.

File: library/src/main/res/values/library_context_menu_android_strings.xml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<resources>
+	<string name="define_int_context_menu.android"></string>
```
Comment (mikepenz) [inline library/src/main/res/values/library_context_menu_android_strings.xml]: still everywhere `define_int_context_menu.android` and not `define_int_context_menu_android`

Comment (axelrindle) [inline library/src/main/res/values/library_context_menu_android_strings.xml]: Oh yeah, sorry xD

**Comment by axelrindle** [comment]
@mikepenz Changed it.

**Comment by mikepenz** [comment]
thanks

**Pull request merged**